faq.adoc

```diff
@@ -60,13 +76,16 @@ Example response
 
 --
 See https://github.com/iZettle/api-documentation/blob/master/authorization.adoc[OAuth] for more info.
+****
+====
 
-'''
-
-=== https://github.com/iZettle/api-documentation/blob/master/product-library.adoc[Product Library]
+== Product Library
+.*_I'm using your API to create Products on iZettle, i have a problem when assigning images to products, i used the image API to create an image and it's working, but how to assign this image to the product i'm creating?_*
```

Comment:
There're many writing typos and wrong styles in the existing docs like this here. I can fix them in another PR.